src/core.js

```diff
@@ -117,15 +117,14 @@ $.extend( $.fn, {
 
 	// http://jqueryvalidation.org/rules/
 	rules: function( command, argument ) {
-
-		// If nothing is selected, return nothing; can't chain anyway
-		if ( !this.length ) {
-			return;
-		}
-
 		var element = this[ 0 ],
 			settings, staticRules, existingRules, data, param, filtered;
 
+		// If nothing is selected, return empty object; can't chain anyway
+		if ( element == null || element.form == null ) {
+			return {};
```

Comment:
I went with an empty object because this method normally returns an object.  I called this out in my bug report (#1770), and I don't feel strongly about `{}` over `undefined`, but I think keeping the return type consistent is reasonable.